training/provision/playbook.yml

```diff
@@ -15,16 +15,51 @@
   - name: Gather facts for first time
     ansible.builtin.setup:
 
-  - name: Check Podman Present
-    ansible.builtin.package:
-      name: podman
+  - name: Required packages
+    ansible.builtin.dnf:
+      name:
+      - https://s3.eu-west-2.amazonaws.com/amazon-ssm-eu-west-2/latest/linux_amd64/amazon-ssm-agent.rpm
+      - podman
       state: present
+      disable_gpg_check: true
 
-  - name: Bootc install
+  - name: Derived Image Containerfile
+    ansible.builtin.template:
+      src: ./templates/Containerfile.j2
+      dest: /tmp/Containerfile
+
+  - name: Login to default registry
+    containers.podman.podman_login:
+      username: "{{ registry_user }}"
+      password: "{{ registry_password }}"
+      registry: quay.io
+      authfile: /etc/containers/auth.json
+
+  - name: Build Derived Image
+    async: 1000
+    poll: 0
+    register: build_result
+    containers.podman.podman_image:
+      name: "quay.io/ai-lab/{{ image_name }}:latest"
+      build:
+        target: localhost/derivied_image:latest
```

Comment:
Typo "derived"

Reply:
But um...actually looking at the ansible docs for this module I see
https://docs.ansible.com/ansible/latest/collections/containers/podman/podman_image_module.html#parameter-build/target
"Specify the target build stage to build." which doesn't sound like what's desired here...

I think what you want is to use what you have in `target` for `name` actually. 

Listen so...I like Ansible, I think it's a powerful swiss army knife that absolutely has its uses.

But this specific code, it doesn't seem like the ansible abstractions are buying us anything over plain old basic shell script executing `podman build` + `podman run`...